source/Track.h

```diff
@@ -0,0 +1,64 @@
+/* Track.h
+Copyright (c) 2022 by RisingLeaf
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef TRACK_H
+#define TRACK_H
```

Comment:
```suggestion
#ifndef TRACK_H_
#define TRACK_H_
```

Reply:
What’s the purpose of the trailing underscore?

Reply:
doesn't really matter, but in every other header it's `CLASS_NAME_H_` with an underscore at the end